users/models.py

```diff
@@ -27,6 +28,12 @@ class User(models.Model):
 
```

Comment:
please add isinstance function to Tean class as you did in the user class to avoid duplicate name

Reply:
I don't exactly understand what do you mean

Reply:
I think the idea is to have a unique team name without the option to add a duplicate team name.
but maybe it will be better to open a new issue about it because it should be in the Team model which is not in the scope of this PR.

an optional solution could be to add unique=True like:
    name = models.CharField(max_length=255, blank=False, unique=True)

for more information please look at: 
https://docs.djangoproject.com/en/1.11/ref/models/fields/#unique

Reply:
opened an issue about it